### DISCUSSION TITLE
p/javaversion/

### DISCUSSION THREAD

**Discussion by giscus** [GenshinMinecraft/GMBlog#18] [Announcements]
# p/javaversion/

妈妈再也不用担心我的McServer开不起来！

https://blog.yzzz.tech/p/javaversion/

**Comment by GenshinMinecraft** [comment]
为什么我们的 Jvav 博士会在这里？

**Comment by GenshinMinecraft** [comment]
哪位大哥给评论区绑到 High Ping Blog 来了...
